tests/waku_core/topics/test_sharding.nim

```diff
@@ -148,6 +148,23 @@ suite "Autosharding":
       check:
         topicMap.error() == "Cannot autoshard content topic: Generation > 0 are not supported yet"
 
-    xtest "catchable error on add to topicMap":
-      # TODO: Trigger a CatchableError or mock
-      discard
+    test "catchable error on add to topicMap":
+      # Given the sequence.add function returns a CatchableError
+      let backup = system.add
```

Comment:
This is used to keep a reference to the original function, which is necessary for cleanup later. 
Produces this error: `Error: invalid type: 'None' for let`